Taskfile.yml

```diff
@@ -203,6 +203,7 @@ tasks:
   reexecute-cchain-range:
     desc: Re-execute a range of C-Chain blocks.
     vars:
+      TASK_NAME: '{{.TASK_NAME | default .TASK}}'
```

Comment:
Should we use `TASK_NAME` or `TASK` consistently and then define a real fallback value here rather than falling back to a different variable?